src/bluehawk/actions/ConsoleActionReporter.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ActionReporter,
+  BluehawkErrorsEvent,
+  FileErrorEvent,
+  FileEvent,
+  FileParsedEvent,
+  FileWrittenEvent,
+  IdsUnusedEvent,
+  ParserNotFoundEvent,
+  StateNotFoundEvent,
+  StatesFoundEvent,
+  WriteFailedEvent,
+} from "./ActionReporter";
+
+export class ConsoleActionReporter implements ActionReporter {
+  _count = {
+    binaryFiles: 0,
+    textFiles: 0,
+    filesWritten: 0,
+    errors: 0,
+  };
+
+  get errorCount(): number {
+    return this._count.errors;
+  }
+
+  onBinaryFile(event: FileEvent): void {
+    ++this._count.binaryFiles;
+    console.log(`found binary file: ${event.sourcePath}`);
+  }
+  onFileParsed(event: FileParsedEvent): void {
+    ++this._count.textFiles;
+    console.log(`parsed file: ${event.sourcePath}`);
```

Comment:
Always logging a line for each event would probably mean very verbose logs, especially if we run commands on a directory.

In most cases I think it would suffice to see only the output of `summary()`.

Any thoughts on a configurable reporter? Maybe we could configure it to log specific events or set a general verbosity level.